### ISSUE TITLE
health check returns 'Object reference not set to an instance of an object'

### ISSUE THREAD

**Issue by svanegmond** [NethermindEth/nethermind#3873]
We are running 1.12.5.    The health checks are enabled in the config
```
  "HealthChecks": {
    "Enabled": true,
    "UIEnabled": false,
    "PollingInterval": 10,
    "Slug": "/health"
  },
```

The health check now returns an error in the description.  See https://rpc.testnet.shyft.network/health

`{"status":"Unhealthy","totalDuration":"00:00:00.0003823","entries":{"node-health":{"data":{},"description":"Object reference not set to an instance of an object.","duration":"00:00:00.0001166","exception":"Object reference not set to an instance of an object.","status":"Unhealthy","tags":[]}}}
`

**Comment by LukaszRozmej** [comment]
@svanegmond are you mining/sealing blocks on this node? Can you share the whole message of the exception that should be in the logs?

**Comment by svanegmond** [comment]
Yes, @LukaszRozmej, this is an AuRA sealer.   I turned debugging all the way up to Trace level and do not obtain any exceptions in the prodigious amounts of log output.

**Comment by svanegmond** [comment]
I tried the 1.12.6 release that was just cut, and that fixed it.

**Comment by LukaszRozmej** [comment]
Yes there was issue with startup order for block producers (because health-check is a plugin), was fixed in 1.12.6
